globals.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '*.svg' {
```

Comment:
What is the module for? Is it to turn the SVGs into React Components?

Reply:
yes, we needed to overwrite the behaviour to use it as a component and not as a string (path) to be used as an image src. Anyway, i removed this usage at the end and changed the approach to load the svgs.